jubatus/server/common/mprpc/rpc_mclient.hpp

```diff
@@ -127,10 +127,8 @@ class rpc_mclient : jubatus::util::lang::noncopyable {
   }
 
   ~rpc_mclient() {
-    if (pool_allocated_ && pool_) {
```

Comment:
``if (pool_allocated_)`` is not necessary?

I think there is a possibility that ``_pool`` is allocated by other object.
https://github.com/jubatus/jubatus/blob/605c70fa42e041619c5f9e3ca4f3742ca8719cac/jubatus/server/common/mprpc/rpc_mclient.hpp#L167-L168

Reply:
You're right!
My pull request was a mistake. Therefore this PR close.